### PR TITLE
fix: Change semantic-release branch from 'master' to 'main'

### PR DESCRIPTION
**Pull request by MarquisLP**
[BrightspaceUI/actions recently renamed its 'master' branch to 'main'](https://github.com/BrightspaceUI/actions).

Our release workflow still pointed to master, which was causing release builds to fail.

### CODE REVIEW

**Review by qwangd2l — APPROVED** [review]

File: capture/d2l-capture-producer/README.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ mocha './test/**/*.visual-diff.js' -t 10000 --golden
 
 > TL;DR: Commits prefixed with `fix:` and `feat:` will trigger patch and minor releases when merged to `master`. Read on for more details...
 
-The [sematic-release GitHub Action](https://github.com/BrightspaceUI/actions/tree/master/semantic-release) is called from the `release.yml` GitHub Action workflow to handle version changes and releasing.
+The [sematic-release GitHub Action](https://github.com/BrightspaceUI/actions/tree/main/semantic-release) is called from the `release.yml` GitHub Action workflow to handle version changes and releasing.
```
Comment (AdmiralCoco) [inline capture/d2l-capture-producer/README.md:194]: Nit: while you're updating this can you also update `sematic-release` -> `semantic release`?

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ npm test
 
 > TL;DR: Commits prefixed with `fix:` and `feat:` will trigger patch and minor releases when merged to `main`. Read on for more details...
 
-The [sematic-release GitHub Action](https://github.com/BrightspaceUI/actions/tree/master/semantic-release) is called from the `release.yml` GitHub Action workflow to handle version changes and releasing.
+The [sematic-release GitHub Action](https://github.com/BrightspaceUI/actions/tree/main/semantic-release) is called from the `release.yml` GitHub Action workflow to handle version changes and releasing.
```
Comment (AdmiralCoco) [inline README.md:47]: Ditto here

**Review by AdmiralCoco — APPROVED** [review]
